app/javascript/app/acuant/document_capture.js

```diff
@@ -50,7 +50,7 @@ export const loadAndInitializeAcuantSdk = () => {
   window.onAcuantSdkLoaded = initializeAcuantSdk;
 
   const sdk = document.createElement('script');
-  sdk.src = '11.4.1/AcuantJavascriptWebSdk.min.js';
+  sdk.src = '/AcuantJavascriptWebSdk.min.js';
```

Comment:
should we rename the files to have any sort of versioning?

Reply:
> should we rename the files to have any sort of versioning?

At least for how we'd needed this in #4128 to bust whichever cached earlier versions may have existed, I'd expect it's quite unlikely someone would have cached the root-level static resource, so it should be fine.

On the other hand, it'd be a kindness to our future selves if we did establish a pattern for versioning that we could leverage for future version upgrades.

Reply:
Based on the findings in the original comment, the SDK is responsible for loading the `.js.mem` file, and does so using a static reference of `AcuantImageProcessingService.js.mem`. So in order to version properly, we'd need it to be more than just the name of the file, but also folder containing the file.

We could do something like `public/acuant/11.4.1/`. We'd need to update this loading behavior:

https://github.com/18F/identity-idp/blob/21102e6cb3dee8a2beb7599525002b96345ff510/app/controllers/acuant_sdk_controller.rb#L20

... probably with a constant reference to the "current" version in the controller.

Reply:
>We could do something like `public/acuant/11.4.1/`

Updated in 5fe4aaa.